vent/common/values.py

```diff
@@ -94,8 +115,9 @@ def safe_range(self) -> tuple:
 
     @safe_range.setter
     def safe_range(self, safe_range):
-        assert(isinstance(safe_range, tuple) or isinstance(safe_range, list))
-        assert(all([isinstance(x, int) or isinstance(x, float) for x in safe_range]))
+        assert(isinstance(safe_range, tuple) or isinstance(safe_range, list) or safe_range is None)
```

Comment:
Should we have asserts in values.py? Or is this for testing?

Reply:
these should fail if an incorrect value is given, they don't have to be asserts but they should be strong exceptions because having values incorrectly specified would fuck up like the whole program